osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -351,14 +367,14 @@ public ModSelectOverlay()
                                                 Right = 20
                                             }
                                         },
-                                        new OsuSpriteText
+                                        ScoreLabel = new OsuSpriteText
                                         {
-                                            Text = @"Score Multiplier: ",

```

Comment:
why did you move this to `updateMods()`?

Reply:
Shouldn't `Score Multiplier:` change color with the multiplier like on stable?

Reply:
No it shouldn't (personal opinion). I think the way it looks in lazer feels cleaner.

Reply:
What does color have to do with the text of a label?